### PR TITLE
Counter error

### PR DESCRIPTION
**Pull request by riquet28**
#### :pushpin: Related Issues
- Related to #?
- Fixes #?

#### :clipboard: Subtasks
- [x] Subtask 1
- [ ] Subtask 2

### :camera: Screenshots (optional)
Description

### CODE REVIEW

**Comment by juliesimon** [comment]
Hello @riquet28 
@moustachu and I looked at your code and even though we're sure your solution works, it still doesn't tell us why we have this bug in French but not this bug in the other languages : 
http://decidim-trad-auto.herokuapp.com/processes/totam-dolor/f/29/?locale=nl
If you look at this link, you can see that when there is no proposal, in French it says "1 proposition" when in Nederlands it says "0 voorstellen"
What we would like to know is where this bug comes from ! And once we'll know that, we probably find a solution that doesn't include adding another locale "no_proposal"

Could you please keep looking at it ?
Thanks !

**Comment by riquet28** [comment]
@juliesimon @moustachu , I think the problem of the bug is detailed here `https://github.com/Locale/localeapp/wiki/Create-Pluralised-Translations` ... it is due to the pluralization ... I will try to fix the problem otherwise

**Comment by juliesimon** [comment]
Thanks @riquet28 !

File: decidim-core/app/validators/etiquette_validator.rb [hunk 1]
```diff
@@ -6,8 +6,8 @@ class EtiquetteValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     return if value.blank?
 
-    validate_caps(record, attribute, value)
-    validate_marks(record, attribute, value)
+    # validate_caps(record, attribute, value)
+    # validate_marks(record, attribute, value)
```
Comment (ludivinecp) [inline decidim-core/app/validators/etiquette_validator.rb]: C'était quoi le problème du coup ? Pourquoi tu as du commenter ces 2 méthodes ? (Juste pour être sûre que ça n'aura pas d'incidence autre part)

Comment (ludivinecp) [reply decidim-core/app/validators/etiquette_validator.rb]: Il me semble qu'elles étaient déjà commenté en 0.10, c'est bizarre que tu ais eu à les recommenter

**Pull request merged**